docs/getting-started-step-by-step/01-setup.md

```diff
@@ -0,0 +1,92 @@
+# Setup: Setting up Crossplane on Kubernetes.
+
+The goal in this part is to install the Crossplane Runtime and necessary tooling to get started:
+
+* A) Access a Kubernetes cluster 
+* B) Install the Crossplane CLI
+* C) Install Crossplane Runtime
+
+<details><summary>Prerequisites</summary>
+
+You should bring basic Kubernetes and cloud provider knwowlege as we will use Crossplane to connect both. Also please make sure you have [kubectl](https://kubernetes.io/docs/tasks/tools/#kubectl) and [Helm](https://helm.sh/docs/intro/install/) installed. If you don't have access to a Kubernetes cluster we recommend [kind](https://kind.sigs.k8s.io/docs/user/quick-start#installation) for setting up a local cluster.
+</details>
+
+## A) Access a Kubernetes cluster
+
+Crossplane runs on any Kubernetes cluster you'd like. If you do not have one handy, feel free to create a local cluster:
+
+### Run:
+```bash
+# Install a local https://kind.sigs.k8s.io cluster:
+$ kind create cluster --image kindest/node:v1.16.15 --wait 5m
```

Comment:
Style nit: although I like how it looks I believe we've aligned on omitting the proceeding `$` to make it possible to copy-and-paste all of our example commands without needing to dodge the `$` symbols.

Reply:
:- ) I was going back and forth on this. I choose readability over interaction at this point. The perfect combination would be if we had a  Copy button that just takes the command without the shell prompt.

Reply:
Similarly to wrapping lines, readability is quite subjective here. I'm also historically in the "pro $" camp, but we've debated this in the past and landed where we're at for the aforementioned reasons. Unless we can agree that including the $ is objectively an improvement rather than a personal preference I'd like us to treat this like style preferences while programming and stick with the convention of the existing files.

Reply:
The reason for the $ is increasing the readability as it clearly indicates a command. Note that we are using this in our docs today. e.g. if we want to separate the command from the result: https://crossplane.io/docs/v1.3/getting-started/provision-infrastructure.html

Reply:
As this has been discussed earlier I've added it as a task and will resolve this conversationtion. But I think would have this in a style guide which can be challenged at a later point.